src/elements/StackableBorderBox.tsx

```diff
@@ -0,0 +1,16 @@
+import { BorderBox } from "./BorderBox"
+
+/**
+ * A stackable border box is a BorderBox that shares borders with its siblings.
+ */
+export const StackableBorderBox = BorderBox.extend`
+  :not(:first-child) {
+    border-top-left-radius: 0;
+    border-top-right-radius: 0;
+  }
+  :not(:last-child) {
+    border-bottom: 0;
+    border-bottom-left-radius: 0;
+    border-bottom-right-radius: 0;
+  }
```

Comment:
@ds300 - note that we're missing the `media` helper here. I have a question out to @zephraph about how we're using the one in `helpers`.